### PR TITLE
adapted new_cluster to create workers in specified region and shutdown_on_close to coiled_params

### PR DESCRIPTION
**Pull request by TejasMorbagal**
[Description of PR]

This PR modifies `new_cluster` to create workers in specified region, by default 'eu-central-1' and adds parameter `shutdown_on_close` to coiled_params

Checklist:

* [x] Add unit tests and/or doctests in docstrings
* [ ] Add docstrings and API docs for any new/modified user-facing classes and functions
* [ ] New/modified features documented in `docs/source/*`
* [x] Changes documented in `CHANGES.md`
* [x] GitHub CI passes
* [x] AppVeyor CI passes
* [x] Test coverage remains or increases (target 100%)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -19,6 +19,15 @@
   ...
   ```
 
+### Fixes
+
+* Added `shutdown_on_close=True` parameter to coiled params to ensure that the 
+  clusters are shut down on close. (#881)
+* Introduced new parameter `region` for utility function `new_cluster` in 
+  `xcube.util.dask` which will ensure coiled creates the dask cluster in the 
+  specified region. (#882)
```
Comment (AliceBalfanz) [inline CHANGES.md:28]:
```suggestion
  prefered default region: eu-central-1. (#882)
```

File: CHANGES.md [hunk 2]
```diff
@@ -19,6 +19,15 @@
   ...
   ```
 
+### Fixes
+
+* Added `shutdown_on_close=True` parameter to coiled params to ensure that the 
+  clusters are shut down on close. (#881)
+* Introduced new parameter `region` for utility function `new_cluster` in
```
Comment (AliceBalfanz) [inline CHANGES.md:26]: I think this entry should rather not be listed under fixes, as it was not a problem but we want to ensure that per default our prefered region is used.

**Review by AliceBalfanz — APPROVED** [review]
Minor change suggestions. Otherwise I am happy :) Thanks for the adjustments!

File: CHANGES.md (hunk 2 above)
Comment (TejasMorbagal) [reply CHANGES.md:26]: Hmm, yeah, I agree.  I have listed them under other changes. Fine?

**Review by pont-us — APPROVED** [review]
Looks good to me!

File: CHANGES.md (hunk 2 above)
Comment (AliceBalfanz) [reply CHANGES.md:26]: Yes, fine with this :)

**Pull request merged**